src/utils/emoji.js

```diff
@@ -10,10 +10,11 @@ const regex = emojiRegex();
 
 const reflect = promise => (...args) => promise(...args).then(v => v, e => e);
 
-const fetchEmojiImage = reflect(resolveImage);
+const makeFetchEmojiImage = () => reflect(resolveImage);
```

Comment:
Can you explain the reason of this change?

Reply:
When it is a constant, it is not possible to mock resolveImage as it uses the real version before jest is able to mock it. I will add a comment.